esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -1359,6 +1376,8 @@ bool CWsWorkunitsEx::onWUGetDependancyTrees(IEspContext& context, IEspWUGetDepen
 {
     try
     {
+        ensureWsCreateWorkunitAccess(context);
```

Comment:
Was this permission never checked before?

Reply:
It was checked at line 1414. I am moving it here.